### PR TITLE
feat(SplitViewWindows): Adds SplitViewWindows view manager

### LINKED ISSUE #184: Add ReactDrawerLayoutManager

### PR DESCRIPTION
**Pull request by rozele**
SplitViewWindows is a similar control to the DrawerLayoutAndroid, it's a pull-out context menu. Some of the events and properties are different, so we went for a different component name altogether.

Fixes #184

### CODE REVIEW

File: ReactWindows/Playground/AppReactPage.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ public override string MainComponentName
         {
             get
             {
-                return "ReactRoot";
+                return "UIExplorerApp";
```
Comment (rozele) [inline ReactWindows/Playground/AppReactPage.cs]: Revert this.

File: Examples/UIExplorer/UIExplorerApp.windows.js [hunk 2]
```diff
@@ -33,8 +33,9 @@ const UIExplorerExampleList = require('./UIExplorerExampleListWindows');
 const UIExplorerList = require('./UIExplorerList');
 const UIExplorerNavigationReducer = require('./UIExplorerNavigationReducer');
 const UIExplorerStateTitleMap = require('./UIExplorerStateTitleMap');
+const UIExplorerHeaderWindows = require('./UIExplorerHeaderWindows');
 
-var DRAWER_WIDTH_LEFT = 56;
+var PANE_WIDTH = 250;
```
Comment (mpodwysocki) [inline Examples/UIExplorer/UIExplorerApp.windows.js]: Weird to have this as a var when it should be a constant value aka `const`

File: ReactWindows/Playground/AppReactPage.cs [hunk 3]
```diff
@@ -19,7 +19,7 @@ public override string JavaScriptMainModuleName
         {
             get
             {
-                return "ReactWindows/Playground/index.windows";
+                return "Examples/UIExplorer/UIExplorerApp.windows";
```
Comment (mpodwysocki) [inline ReactWindows/Playground/AppReactPage.cs]: Please revert

File: ReactWindows/ReactNative/UIManager/UIManagerModule.Constants.cs [hunk 4]
```diff
@@ -229,8 +229,7 @@ public partial class UIManagerModule
                             "window",
                             new Dictionary<string, object>
                             {
-                                { "width", 100 },
-                                { "height", 100 },
+                                /* TODO: handle variable height/width? */
```
Comment (mpodwysocki) [inline ReactWindows/ReactNative/UIManager/UIManagerModule.Constants.cs]: Can we file an issue for this?

**Pull request merged**